ONSdigital/census-rm-qid-batch-runner
# RMC-141 Unaddressed & CCS print file exporter

# Motivation and Context
Add a script to generate the unaddressed QIC/UAC pairs print files.

# What has changed
A new script has been added.
A new Pod description has been added to run the script from.

# How to test?
To test running in kubernetes, before following the README instructions you'll need to build and push up a version of the image from this branch for kubernetes to pull
Check out branch.
Build Docker image locally and push
```bash
docker build . -t eu.gcr.io/<your-project-id>/census-rm-qid-batch-runner:latest
docker push eu.gcr.io/<your-project-id>/census-rm-qid-batch-runner:latest
```

Edit the qid-batch-runner.yml and point it to the image you just pushed, eu.gcr.io/<your-project-id>/census-rm-qid-batch-runner:latest

Then follow the README instructions for running in kubernetes

__NOTE:__ If you have any existing unaddressed qid pairs in your database the print file generation will fail. Either start with a clean database or delete all the rows from the `uac_qid_link` table where `caze_case_ref` is `NULL`.

# Links
https://trello.com/c/tFx0hjmx/646-rmc-141-unaddressed-ccs-print-file-exporter-13
